libs/blockscout-service-launcher/src/settings.rs

```diff
@@ -58,17 +58,28 @@ impl Default for MetricsSettings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum TracingFormat {
```

Comment:
I think it's better to call this log format

Reply:
this enum in setting `Tracing`, it will be used by `tracing.rs` file only, also we set Json format for `tracing_sunscriber`, so I think it's better to leave it as TracingFormat